doc/ref_model/chapters/appendix-a.md

```diff
@@ -45,25 +53,48 @@ Several specific technology areas have been identified by CNTT as using an ABI i
 
 The preceding list is not exhaustive; technologies will be added as required.
 
-Current CNTT Policies:
+<a name="A.5.3"></a>
+### A.5.3 Current CNTT Policies
+
+The following sets of compromises and transition plans comprise the policy for each technology subject to this appendix.
+
+Be aware the compromises and transition plans contained herein, are directly related to factors which are subject to change with the evolution of technology, with changes in industry direction, with changes in standards, etc. Hence, **the policies are subject to change without notice**, and the reader is advised to consult the latest online Github revision of this chapter. All locally stored, printed or other copies should be considered obsolete.
+
+> **Note to Authors:** Status should be set to "Proposed" when initial content is entered. Once alignment is attained following vetting and discussion, status should be set to "Aligned". Immediately prior to merge, status should be set to "In Force". When amending previously approved language, status should be changed from "In Force" to "In Force (Pending Changes)", followed by "Aligned" and ultimately, "In Force".
+
+**SR-IOV**
+- **Status:** Proposed
+- **Revision:** v0.1 - 8/12/2019 - Shostak
+
+SR-IOV transgresses the CNTT Abstraction Principle, which prohibits hardware-dependent code be required in a workload. Refer to the Abstraction Principle in RM Chapter 1 for the related rationale and details. However, CNTT recognizes that today, SR-IOV provides a capability to increase throughput over network interfaces at an economical cost, that some applications as currently implemented, are highly dependent upon. As such, the CNTT approach to SR-IOV shall be:
+
+> **DISCLAIMER: I (Mark) am typing this proposed language based on input I have received. Therefore, people with strong views may need to discuss this online or live, and come to a consensus. In other words, this is a starting point. Expect there to be discussion and don't shoot the messenger.** (To be removed before merge)
```

Comment:
Generally speaking, I like the policy framework being proposed.  How would we be best served for discussing the language of each of the policies in this section?  Should we break out each one into its own individual github issue so we can discuss freely, while allowing this framework to be merged? discuss in-line here?  other approaches?

Reply:
@kedmison Hi again, Yes, that's exactly what I was envisioning; I was trying to decouple them from each other and from the process, so individual tech discussions could follow a uniform process, and focus on the tech, not how or where to put it in the doc. WRT SRIOV, I'm thinking whether we need to decouple will become apparent, shortly.